quantmsrescore/ms2rescore.py

```
@@ -2,313 +2,193 @@
 # This script is part of the quantmsutils package
 
-import importlib.resources
-import json
 import logging
 import os
+from typing import List
+
 import click
 import pyopenms as oms
-from ms2rescore import package_data, rescore
+from ms2rescore import rescore
```

Ruff (0.8.2)

`ms2rescore.rescore` imported but unused

Remove unused import: `ms2rescore.rescore`

(F401)